.goreleaser.yml

```diff
@@ -0,0 +1,59 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+- env:
+  - CGO_ENABLED=0
+  goos:
+    - freebsd
+    - openbsd
+    - solaris
+    - windows
+    - linux
+    - darwin
+  goarch:
+    - amd64
+    - '386'
+    - arm
+    - arm64
+  ignore:
+    - goos: darwin
+      goarch: '386'
+    - goos: darwin
+      goarch: arm
+    - goos: darwin
+      goarch: arm64
+    - goos: openbsd
+      goarch: arm
+    - goos: openbsd
+      goarch: arm64
+    - goos: solaris
+      goarch: arm
+    - goos: solaris
+      goarch: arm64
+    - goos: solaris
+      goarch: '386'
+    - goos: windows
+      goarch: arm
+    - goos: windows
+      goarch: arm64
+  binary: '{{ .ProjectName }}_{{ .Tag }}'
+archives:
+- format: zip
+  name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
+checksum:
+  name_template: '{{ .ProjectName }}_{{ .Version }}_SHA256SUMS'
+  algorithm: sha256
+signs:
+  - artifacts: checksum
+    args:
+      - "--local-user"
+      - "51852D87348FFC4C" # Replace this with your GPG signing key ID
+      - "--output"
+      - "${signature}"
+      - "--detach-sign"
+      - "${artifact}"
+release:
+  draft: true
```

Comment:
We may be able to remove this line as well? We don't need to force the draft, and they can customize it?

Reply:
Perhaps! Right now I've been testing publishing with draft -> assets -> published release. I have not yet tested the publishing behavior with a possible race condition around creating a finalize release and uploading assets.

Reply:
Oh interesting, then I'd say leave it but comment in here that they need to finalize publish manually in the GitHub UI for the registry to pick it up?

Reply:
Done!